tools/submission/submission_checker.py

```diff
@@ -1949,6 +1949,12 @@ def log_result(
                 "Offline": "Tokens/s",
                 "Server": "Tokens/s",
             },
+            "llama3.1-405b": {
+                "SingleStream": "Latency (ms)",
```

Comment:
Do we need to remove Singlestream and Multistream to avoid confusion?

Reply:
Sure, will do